bridge-impl/src/main/java/com/liferay/faces/bridge/renderkit/bridge/internal/ResponseWriterBridgeCompat_2_0_Impl.java

```diff
@@ -48,6 +48,35 @@
 	protected static final String VIEW_STATE_MARKER = PartialResponseWriter.VIEW_STATE_MARKER;
 	protected static final String XML_MARKER = "<?xml";
 
+	// Private Constants
+	private static final boolean JSF_RUNTIME_SUPPORTS_NAMESPACING_VIEWSTATE;
+
+	static {
+
+		boolean namespacedViewStateSupported = false;
```

Comment:
@ngriffin7a, I changed this default to `false`. That seemed more correct to me, but please review to ensure that is correct.